nicolegooden/ideabox-boilerplate
# Add gifs and screenshots

What’s this PR do?
- Add gifs and screenshots for README.md

  Where should the reviewer start?  
- N/A

How should this be manually tested?  
- Make sure the pictures/gifs load properly

Any background context you want to provide?  
- Cameron helped me out with the video style and added a little artistic style

What are the relevant tickets?  
- N/A

Screenshots (if appropriate)
- N/A

  Questions:
- How do you create a directory in github's gui?
